packages/primitives/src/Button.tsx

```diff
@@ -182,16 +182,37 @@ export const iconButtonRecipe = cva({
   base: {
     lineHeight: "1",
     minHeight: "unset",
-    height: "xxlarge",
-    width: "xxlarge",
-    "& svg": {
-      marginInline: "0",
-      marginBlock: "0",
-      width: "medium",
-      height: "medium",
+  },
+  defaultVariants: {
+    size: "default",
+  },
+  variants: {
+    size: {
+      default: {
```

Comment:
Har angret litt på at jeg kalte denne `default` for vanlig knapp. Tenker vi kanskje heller skal kalle den medium?